### DISCUSSION TITLE
publish an application in playsore

### DISCUSSION THREAD

**Discussion by ghost** [airsdk/Adobe-Runtime-Support#3326] [Ideas]
I have an application coded with actionscript 3 in adobe animate 2024. How i can publishing it to android and ios and how to upload them to playstore help me please to do it.

**Comment by ajwfrost** [comment]
Hi

For Android, there is a tutorial available at https://airsdk.dev/docs/tutorials/platform/android/packaging-android-app-bundles

@marchbold wondering if it's possible to get a similar tutorial for iPhone/iPad apps: https://airsdk.dev/docs/tutorials/platform/ios/packaging-ipa is a little lacking!

thanks

**Reply by ghost** [reply]
i can publish the application in adobe animate for 32bit or 64 bit but splash screen appear in starting application, is playstore accept this or no.

**Reply by marchbold** [reply]
@ajwfrost Sure thing! I'll put together a tutorial for iOS and put it up in the coming weeks.

**Reply by 2jfw** [reply]
> @ajwfrost Sure thing! I'll put together a tutorial for iOS and put it up in the coming weeks.

Great thing , this will surely help a lot!

I always found publishing to iOS/App Store very verbose and complicated, especially the part where you need to use a Mac when developing on Windows, in order to apply the provisioning profile and such.
(I know there is or was an online service for this but you would need to provide them more or less confidential data of your app, at least as far as I remember...)
